src/constructs/ecs/ecs-task.ts

```diff
@@ -145,7 +145,7 @@ export class GuEcsTask {
       // see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html#cfn-ecs-taskdefinition-cpu for details
       cpu = 2048, // 2 cores and from 4-16GB memory
       memory = 4096, // 4GB
-      storage = 20, //20 GB
+      storage = 21, //20 GB
```

Comment:
The comment is out of sync now.

Reply:
> I'd suggest _not_ defaulting `storage`, allowing AWS CDK to handle that. Instead, I'd validate `storage` if set and throw when < 21:

Great idea, have pushed a commit to do this, and tested on Lurch CODE

Reply:
For completeness, probably want to throw when >200 too.

Reply:
My recommendation would be to leave the validation to aws-ecs' `TaskDefinition`:

https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs/lib/fargate/fargate-task-definition.ts#L143-L145

If those ranges change over time, this code will be out of sync. You should be able to unit test that passing invalid values like 20 or 201 already throws an error when creating the `TaskDefinition`.